react-stopwatch-component/src/stopwatch.jsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+
+class StopWatch extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = { time: 0, playing: false };
+    this.handleTimer = null;
+    this.handleClick = this.handleClick.bind(this);
+    this.resetState = this.resetState.bind(this);
+  }
+
+  handleClick() {
+    if (this.state.time > 1 && !this.state.playing) {
+      this.setState({ time: 0, playing: false });
+    } else {
+      this.setState({ time: this.state.time + 1, playing: !this.state.playing });
+    }
+    // this.setState({ time: this.state.time + 1, playing: !this.state.playing });
+
```

Comment:
Here you can use this logic (might need some adjustment) to clear the interval or call `handleCounter` to start the timer running.